PostHog/PostHogStorage.swift

```diff
@@ -15,7 +15,12 @@ import Foundation
  */
 func applicationSupportDirectoryURL() -> URL {
     let url = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first!
-    return url.appendingPathComponent(Bundle.main.bundleIdentifier!)
+    #if canImport(XCTest) // only visible to test targets
+        return url.appendingPathComponent(Bundle.main.bundleIdentifier ?? "com.posthog.test")
+    #else
+        // TODO: Should we be using a fallback temp directory instead of force unwrapping here?
```

Comment:
If for some reason this can be nil, I'd fall back to in-memory storage but we don't have this yet.
We cannot cache everything in the applicationSupportDirectory that is visible for all the apps, this has to be sandboxed for only this app.